### DISCUSSION TITLE
请问全局字体中英文字母与中文的间隔过大该如何调整？

### DISCUSSION THREAD

**Discussion by Nemo137-Liang** [CaiJimmy/hugo-theme-stack#671] [Q&A]
发现全局的字体显示中，英文字母与相邻的中文之间的间隔会很大，中文与中文、字母与字母之间则不会出现该问题。另尝试使用其他字体依然无法解决，因此特来请教，该如何调整英文字母与中文之间的间隔？

![image](https://user-images.githubusercontent.com/60751406/187952350-f12282fe-5f91-4aa2-afb1-9647f9357b0d.png)

**Comment by heykitin** [comment]
中文和英文字母本就应该隔开
